test/extended/machines/cluster.go

```diff
@@ -120,83 +122,169 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(nodes.Items).NotTo(o.HaveLen(0))
 
-		expected := make(map[string]int)
-		actual := make(map[string]int)
+		bootTimelinesByNode := make(map[string][]bootTimelineEntry)
 		errs := make([]error, 0)
 
+		allNodeLogs := &bytes.Buffer{}
 		// List nodes, set actual and expected number of reboots
 		for _, node := range nodes.Items {
-			// Examine only the nodes which are available at the start of the test
-			if !slices.Contains(staticNodeNames, node.Name) {
```

Comment:
this check resulted in zero nodes, because `openshift-tests` starts a new process to run an individual test in, so the tests must be self-contained.  The `staticNodeNames` had no values, so no logs were checked, so the test always passed without checking anything